exchanges/request/request_test.go

```diff
@@ -559,77 +513,61 @@ func TestBasicLimiter(t *testing.T) {
 }
 
 func TestEnableDisableRateLimit(t *testing.T) {
-	r, err := New("TestRequest", new(http.Client), WithLimiter(NewBasicRateLimit(time.Minute, 1, 1)))
-	if err != nil {
-		t.Fatal(err)
-	}
-	ctx := context.Background()
+	rl := NewBasicRateLimit(50*time.Millisecond, 1, 1)
+	r, err := New("TestRequest", http.DefaultClient, WithLimiter(rl))
+	require.NoError(t, err, "New requester must not error")
 
-	var resp any
-	err = r.SendPayload(ctx, Auth, func() (*Item, error) {
-		return &Item{
-			Method: http.MethodGet,
-			Path:   testURL,
-			Result: &resp,
-		}, nil
-	}, AuthenticatedRequest)
-	if err != nil {
-		t.Fatal(err)
+	sendIt := func() error {
```

Comment:
🐒 This monkey doesn't like the word `it`
```suggestion
	sendPayload := func() error {
```

Reply:
You're not familiar with the famous send it meme? 😂  Leaving it in